app/models/manageiq/providers/oracle/infra_manager.rb

```diff
@@ -0,0 +1,101 @@
+require 'uri'
+
+class ManageIQ::Providers::Oracle::InfraManager < ManageIQ::Providers::InfraManager
+  require_nested :RefreshWorker
+  require_nested :RefreshParser
+  require_nested :Host
+  require_nested :Refresher
+  require_nested :Template
+  require_nested :Vm
+
+  def self.ems_type
+    @ems_type ||= "oraclevm".freeze
+  end
+
+  def self.description
+    @description ||= "Oracle Virtualization Manager".freeze
+  end
+
+  def self.default_blacklisted_event_names
+    []
+  end
+
+  def supports_port?
+    true
+  end
+
+  def supported_auth_types
+    %w(default)
+  end
+
+  def supports_authentication?(authtype)
+    supported_auth_types.include?(authtype.to_s)
+  end
+
+  def self.raw_connect(server, port, username, password)
+    require 'fog/oracle'
+
+    url = URI::HTTPS.build(:host => server, :port => port).to_s
+
+    params = {
+      :oracle_username => username,
+      :oracle_password => password,
+      :oracle_url      => url,
+      :connection_options => {
+        :ciphers => 'HIGH:RC4:!SSLv2:!aNULL:!eNULL:!3DES',
```

Comment:
this is going to need to change when the list of secure ciphers changes. (which seems to have happened a bunch over the past year or two)

Can we use the default list here?